rust/Cargo.toml

```diff
@@ -19,9 +19,10 @@ sha3 = "0.10.8"
 [dev-dependencies]
 # workaround for enabling mock feature also in integration tests
 evmrs = { path = ".", features = ["mock"] }
+driver = { path = "driver", features = ["mock"] }
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "rlib"]
```

Comment:
`cdylib` is needed to create a dynamic C lib which can be called from Go. The tests need `rlib` because they need type definitions from the main library.